lnwire/channel_update_2.go

```diff
@@ -70,6 +70,11 @@ type ChannelUpdate2 struct {
 	// millionth of a satoshi.
 	FeeProportionalMillionths tlv.RecordT[tlv.TlvType18, uint32]
 
+	// InboundFee is an optional TLV record that contains the fee
+	// information for incoming HTLCs.
+	// TODO(elle): assign normal tlv type?
+	InboundFee tlv.OptionalRecordT[tlv.TlvType55555, Fee]
```

Comment:
I think we can just use `FeeRecordType`.

Reply:
that just defines an int. we still need the tlv type here

Reply:
(same was done in ChannelUpdate1)